File: ApplicationServices/Models/Flights/AirportDto.cs
using ApplicationServices.Model.Country;

namespace ApplicationServices.Models.Flights
{
    public class AirportDto
    {

        public string id { get; set; }
        public string name { get; set; }
        public string detailedName { get; set; }
        public string iataCode { get; set; }
        public CountryDto countryDto { get; set; }
        public int travelersScore { get; set; }
    }
}

File: WebService/src/app/model/airport/airport.dto.ts
import { CountryDto } from "../countries/CountryDto";

export class AirportDto {
  id: string;
  name: string;
  detailedName: string;
  iataCode: string;
  countryDto: CountryDto;
  travelersScore: number;

  constructor(
    id: string,
    name: string,
    detailedName: string,
    iataCode: string,
    countryDto: CountryDto,
    travelersScore: number
  ) {
    this.id = id;
    this.name = name;
    this.detailedName = detailedName;
    this.iataCode = iataCode;
    this.countryDto = countryDto;
    this.travelersScore = travelersScore;
  }
}
